doc/news/README.rst

```diff
@@ -0,0 +1,34 @@
+Recording Changes
+=================
+ 
+This directory contains "news fragments" which are small, structured text files that contain information about changes 
+or updates that will be included in the release notes. 
+These fragments are used to automatically generate changelogs or release notes.
+They can be written restructured text format or plain text.
+
+Each file should be named like ``<JIRA TICKET>.<TYPE>.<EXT>`` with a file extension defining the markup format (``rst|md``)
+The ``<TYPE>`` should be one of:
+
+* ``feature``: A new feature
+* ``bugfix``: A bug fix.
+* ``perf``: A performance enhancement.
+* ``doc``: A documentation improvement.
+* ``removal``: A deprecation or removal of API.
+* ``misc``: Other minor changes and/or additions
+
+An example file name would therefore look like ``DM-40534.doc.rst``.
+    
+You can test how the content will be integrated into the release notes by running ``towncrier build --draft --version=v0.26.1``.
+To produce the news file for real, run:
+
+.. code-block:: bash
+
+   $ towncrier build --version=v0.26.1
+
+Note that you have to run it from the root repository directory (i.e. the ``ts_externalscripts``). 
+``towncrier`` can be installed from PyPI or conda-forge. 
+
+.. note::
+
+   When running towncrier to build the changelog, you may be prompted to confirm the deletion of fragments. 
+   Please do not confirm the deletion, and the fragments will be retained in the doc/news directory.
```

Comment:
I think it is ok to delete the fragments no? Why would we keep them?

Reply:
I saw other repositories keeping it. It was very useful to understand how to create the fragments, so I just thought it would be good to keep them.